aiven/client/argx.py

```diff
@@ -193,50 +200,60 @@ def add_cmds(self, add_func):
             if getattr(func, ARG_LIST_PROP, None) is not None:
                 add_func(func)
 
-    def parse_args(self, args=None):
+    def parse_args(self, args: Optional[Sequence[str]] = None) -> None:
         self.extend_commands(self)
 
         if ARGCOMPLETE_INSTALLED:
             argcomplete.autocomplete(self.parser)
 
-        args = self.parser.parse_args(args=args)
+        ext_args = self.parser.parse_args(args=args)
         for ext in self._extensions:
-            ext.args = args
+            ext.args = ext_args
 
-    def pre_run(self, func):
+    def pre_run(self, func: Callable) -> None:
         """Override in sub-class"""
 
-    def expected_errors(self):
+    def expected_errors(self) -> List[Type[BaseException]]:
         return []
 
+    def _to_mapping_collection(
+        self, obj: Union[Mapping[str, Any], Collection[Mapping[str, Any]]], single_item: bool = False
+    ) -> Collection[Mapping[str, Any]]:
+        if single_item:
+            assert isinstance(obj, Mapping)
+            return [obj]
+        else:
+            assert isinstance(obj, Collection)
+            return cast(Collection[Mapping[str, Any]], obj)
```

Comment:
Is it possible to avoid `cast` here? The `assert` on the previous line suppose to limit the result to only one option.

Reply:
Unfortunately, mypy is not that smart:
```
aiven/client/argx.py:227: error: Incompatible return value type (got "Union[Mapping[str, Any], Collection[Mapping[str, Any]]]", expected "Collection[Mapping[str, Any]]")
```

Reply:
Hopefully we can fix it later